DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -150,6 +150,10 @@ namespace l1t {
     float sigmaPhiPhiTot() const { return sigmaPhiPhiTot_; }
     float sigmaZZ() const { return sigmaZZ_; }
     float sigmaRRTot() const { return sigmaRRTot_; }
+    float varRR() const { return varRR_; }
+    float varZZ() const { return varZZ_; }
+    float varEtaEta() const { return varEtaEta_; }
+    float varPhiPhi() const { return varPhiPhi_; }
```

Comment:
I guess it isn't deal breaking for me, but it seems the difference between _getting_ variables and _setting_ variables in this code is whether or not an argument is provided to this overloaded function. Diametrically opposed functionality of an overloaded function is probably a bit dangerous. A bit more traditional would be to label these methods with `get` and `set` as appropriate.

Reply:
Thanks @aloeliger 
I have prepended all setters in this class with ˋsetˋ to differentiate them from getters.